GramAddict/core/views.py

```diff
@@ -112,8 +117,11 @@ def _navigateTo(self, tab: TabBarTabs):
 
         if button.exists():
             # Two clicks to reset tab content
+            random_sleep(2, 4)
```

Comment:
were these to fix a bug?

Reply:
yes, sometimes the interface has not been loaded